pkgs/development/libraries/openmpi/default.nix

```diff
@@ -40,21 +52,28 @@ in stdenv.mkDerivation rec {
     find -name "Makefile.in" -exec sed -i "s/\`date\`/$ts/" \{} \;
   '';
 
-  buildInputs = with stdenv; [ gfortran zlib ]
+  buildInputs = with stdenv; [ gfortran zlib libfabric pmix ucx ]
     ++ lib.optionals isLinux [ libnl numactl ]
     ++ lib.optionals cudaSupport [ cudatoolkit ]
     ++ [ libevent hwloc ]
     ++ lib.optional (isLinux || isFreeBSD) rdma-core;
 
   nativeBuildInputs = [ perl ];
 
-  configureFlags = with stdenv; lib.optional (!cudaSupport) "--disable-mca-dso"
+  configureFlags = with stdenv; [
+    "--with-cma"
+    "--with-hwloc=${hwloc.dev}"
+    "--with-ofi=${libfabric}"
```

Comment:
What benefit does libfabric bring (I am not familiar with libfabric)?

Reply:
I think OmniPath needs `libfabric` - can't try though, since I have no OmniPath machine in a test setup

Reply:
I would prefer to remove it then.

Reply:
as a dependency, or the whole package?

Reply:
I would leave the whole package out for now since the `libfabric`  is not even built with OmniPath in the present version. We can add that later, when it becomes relevant.